Cargo.toml

```
@@ -14,6 +14,7 @@ rust-version = "1.65"
 [dependencies]
 toml = "0.8"
 embed-resource = "3"
+indexmap = "2.9.0"
```

```suggestion
indexmap = "2"
```
we prefer to give consumers the option to use older versions when possible